.github/workflows/docs.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: docs-build
+
+on:
+  push:
+    branches: [ docs-build ]
```

Comment:
Can we change this to `main` before merge?

Reply:
Yes, we can. When we decide that we don't need anymore the docs-build branch, we can modify the docs.yml as it is required.